src/main/java/net/neoforged/neoforge/common/ModConfigSpec.java

```diff
@@ -1021,7 +1051,7 @@ public static class LongValue extends ConfigValue<Long> implements LongSupplier
         }
 
         @Override
-        protected Long getRaw(Config config, List<String> path, Supplier<Long> defaultSupplier) {
+        public Long getRaw(Config config, List<String> path, Supplier<Long> defaultSupplier) {
```

Comment:
Any other alternatives would expose getters that are even more accessible than the getRaw. This method has 3 more parameters which require much more work to get, no one will use it instead of get.

Reply:
What are the other alternatives?
Have we considered creating some encapsulation mechanism that can be supplied to config UIs?

Reply:
Encapsulation such as? The config ui needs the value "on disk" available, and getRaw is quite precisely what provides you with that. There is no footgun exposed here, the method requires a config instance which cannot be accessed directly from the config value and is only exposed by the spec, the path can be got from the config value, and the default supplier has to be provided as a lambda since there's only a getter that already calls it.

And besides, `getRaw` does exactly what it says. Gets the raw value a config holds. It's used for the cached getter, why wouldn't config uis, server dashboards, whatever use it when the `get` is cached and is no longer usable for that use case as it respects the restart flags.

Reply:
Thinking this through again, I don't think Config UIs need getRaw().

The return type of get() and getRaw() is the same, so there is no difference there. The only difference is that get() is cached. While getRaw() is not. That on the other hand should be no hinderance to any config UI, because the value return by getRaw() is always the same as the value returned by get(). Or am I missing something here?

Reply:
Hmm these flags are of an issue.
There are two aspects to consider:

Reload from disk, and UI.
- In UIs it does not matter.
- But when reloading from disk, it might matter.
  - Have we considered filtering the reload from disk to values that can be updated while the game is running?

Reply:
Okey understood. Still don't think you need the FML change.